Dockerfile.gnoland.release

```diff
@@ -8,6 +8,6 @@ COPY ./gno.land/genesis/genesis_txs.jsonl /gnoroot/gno.land/genesis/genesis_txs.
 
 ENV GNOROOT="/gnoroot/"
 
-EXPOSE 26657 36657
```

Comment:
Don't duplicate this

Reply:
(not obvious from this comment, the following line is `EXPOSE 26657 26657`)

Reply:
It's a mistake, it should be 26656 and 26657, updating